client/blocks/post-likes/docs/example.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PostLikes from '../';
+
+function PostLikesExample() {
+	return (
+		<div>
+			<PostLikes siteId={ 3584907 } postId={ 37769 } />
```

Comment:
TIL that post likes is a public endpoint.  This like block for en.blog still works on a non-a8c test account.